lms/djangoapps/courseware/tests/test_inline_analytics_utils.py

```diff
@@ -78,3 +78,41 @@ def test_multi_responses(self):
         self.assertEquals(response[1].response_type, 'radio')
         self.assertEquals(response[0].message, None)
         self.assertEquals(response[1].message, None)
+        self.assertEquals(response[0].choice_name_list, [])
+        self.assertEquals(response[1].choice_name_list, ['choice_0', 'choice_1', 'choice_2'])
+
+    def test_multi_responses_name_mask(self):
+
+        xml = textwrap.dedent("""\
```

Comment:
I'm not seeing the benefit of that. I gain utility by being able to eyeball the xml and test together as one.